src/utils/constants.ts

```diff
@@ -6,6 +6,9 @@ export const ARNS_NAME_REGEX = new RegExp(
 );
 export const ARNS_TX_ID_REGEX = new RegExp('^[a-zA-Z0-9-_s+]{43}$');
 export const ARNS_TX_ID_ENTRY_REGEX = new RegExp('^[a-zA-Z0-9-_s+]{1,43}$');
+export const STUB_ARWEAVE_TXID = '2yHmORN-N12hM1B2f9-JPMpOfa59qhpsExFCzImrD30'; // arns spec pdf
+export const DEFAULT_ANT_SOURCE_CODE_TX =
```

Comment:
nit: we could also use an ID provided by the arns contract state for this value, but fine for now

Reply:
agreed, ideally for the source code tx as well but thats not setup yet